wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,331 @@
+---
+tags:
+  - history
+  - leaderboard
+  - rank 1
+  - rank #1
+  - rankings
+  - top player
+---
+
+<!-- Maintainers Note: Please refer to the original pull request for more info on sources at https://github.com/ppy/osu-wiki/pull/4720 -->
+
+# osu! Rankings History
+
+The following is a list of players who are recognized to have reached Rank #1 in the [osu! Global Rankings](https://osu.ppy.sh/rankings/osu/performance).
+
+Over the years the osu! global rankings has used three different systems for ranking players:
+
+- [Ranked Score](/wiki/score#ranked-score) (July 2007 – 24 July 2012)
+- [ppv1](/wiki/Performance_points/ppv1) (24 July 2012 – 27 January 2014)
+- [ppv2](/wiki/Performance_points) (27 January 2014 – Present)
+
+Because of this, this list will only include players who reached Rank #1 on the main leaderboard that was used at the time of reaching said rank. *Italicized* dates and numbers indicate that the corresponding player are continuing their reign as osu!'s Rank #1 player either from the previous year or onto the following year.
+
+## 2007
+
+*Note: During the period where Ranked Score was used, Rank #1 would change frequently, thus the earlier listings may use months where exact dates aren't available.*
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) | 2007-10-4 | October 2007 |  |
+| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | October 2007 | 2007-10-11 |  |
```

Comment:
According to https://osu.ppy.sh/community/forums/posts/932 the very first version of Online Ranking was finished on 2007-10-01, the in-game online ranking display was added on the 2007-10-06 and the web Player Ranking page was added on the 2007-10-11. If we assume that https://web.archive.org/web/20071011215722/http://osu.ppy.sh:80/index.php?p=player is *effectively* the player ranking as it went up, we can say that Ivalset is likely to be the first-ever #1 player.

As far as I can tell there are no records of eyup having #1 before or after this capture, the only information we have on them having it is from people mentioning him in relevant threads at some point e.g. https://osu.ppy.sh/community/forums/topics/77231 and https://www.reddit.com/r/osugame/comments/7koexn/osu_history_now_from_2007_and_with_bug_fixes/.

Now I do believe that he did have it at some point given how close he is in the rankings on [2007-10-24](https://web.archive.org/web/20071024063540/http://osu.ppy.sh/index.php?p=player), [2007-11-23](https://web.archive.org/web/20071123172250/http://osu.ppy.sh/index.php?p=player) and [2007-12-24](https://web.archive.org/web/20071224062629/http://osu.ppy.sh/index.php?p=player). 

So the two questions I have are;

1. When did eyup get #1?
2. Should we count peppy?

I pose the question regarding peppy mostly because there isn't really any way of knowing if he had it given that all available info points to him never gaining enough score from when the rankings became public. Personally, I lean on the side of not including peppy but I want to know what other people think.

Reply:
When it comes to peppy I'd say the same logic as all the other reigns applies here: If there are no sources for a reign, and no credible claim that reign happened, then that reign shouldn't be mentioned.

Reply:
I think something like this would be more accurate than what we currently have, I don't particularly *like* this, but without a miracle, I don't think we will find any exact dates for this time.

```
| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-11 | October 2007| [\[1\]](https://web.archive.org/web/20071011215722/http://osu.ppy.sh/index.php?p=player) |
| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | October 2007 | 2007-10-24 |  |
| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-24 | 2007-11-17 | [\[1\]](https://web.archive.org/web/20071024063540/http://osu.ppy.sh/index.php?p=player) |
```

Reply:
@FlayFlayer and I have come to some resolutions that solidify the order for 2007 as suggested above. 

For point 1. [This thread](https://osu.ppy.sh/community/forums/topics/342) provides the most evidence we have that eyup was # 1 at some point in October 2007, while it's not an exact timestamp it's as close as I think we are going to get.

As of such the start of 2007 will become
```
| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-11 | 2007-10-15 | [\[1\]](https://web.archive.org/web/20071011215722/http://osu.ppy.sh/index.php?p=player) |
| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | 2007-10-15 | 2007-10-24 | [\[1\]](https://osu.ppy.sh/community/forums/topics/342) |
| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-24 | 2007-11-17 | [\[1\]](https://web.archive.org/web/20071024063540/http://osu.ppy.sh/index.php?p=player) |
```

For point 2, we will drop peppy from the list due to lack of any actual sources to back up any claim. The closest that we get to justifying peppy is in the thread for [DISCO PRINCE](https://osu.ppy.sh/community/forums/topics/213). Its the first-ever ranked map so it is easy to assume that the first score submitted would be by peppy and as of such he should be the first-ever # 1. BUT the score that is currently submitted at https://osu.ppy.sh/scores/osu/1 doesn't exist, and [peppy's current submitted score]() on DISCO PRINCE is from July 2015. 

From this point the pieces of evidence that we have created a weird chain of events that doesn't hold up to scrutiny, as of such we are using the first saved player ranking on [the Wayback machine](https://web.archive.org/web/20071011215722/http://osu.ppy.sh/index.php?p=player) as a sort of ground zero for player rankings to maintain consistency.